#Predictions List 

int8 n                          		# Number of predictions
Prediction[] predictions                # Predictions Data